### PR TITLE
[Doc] add hardware recommendations

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/13078

Preview looks good:
image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@codelipenghui @Huanli-Meng can you help review? Thanks

**Review by michaeljmarshall — APPROVED** [review]
LGTM. @Anonymitaet - out of curiosity, how are you determining the expected throughput for clusters using this hardware? I ask because this kind of throughput may vary from version to version and based on the configurations of the cluster. While it's not necessary to publish this information, it would be interesting to how the throughput varies for each release of Pulsar.

File: site2/docs/deploy-bare-metal.md [hunk 1]
```diff
@@ -63,30 +63,27 @@ For machines running a bookie and a Pulsar broker, more powerful machines are re
...
-    - Storage: 
-      - Journal: 256GB, PD-SSD
-      - Ledger: 2TB, PD-STANDARD
+A cluster consists of 3 broker nodes, 3 bookie nodes, and 3 ZooKeeper nodes. The following recommendation is suitable for one node.
+
+- The minimum hardware settings (**250 Pulsar topics**)
+   
+   Component | CPU|Memory|Storage|Throughput |Rate
+   |---|---|---|---|---|---
+   Broker|0.2|256 MB|/|Write throughput: 3 MB/s<br><br />Read throughput: 6 MB/s<br><br />|Write rate: 350 entries/s<br><br />Read rate: 650 entries/s
```
Comment (michaeljmarshall) [inline site2/docs/deploy-bare-metal.md:76]: Nit:

```suggestion
   Broker|0.2|256 MB|/|Write throughput: 3 MB/s<br><br />Read throughput: 6 MB/s<br><br />|Write rate: 350 messages/s<br><br />Read rate: 650 messages/s
```

File: site2/docs/deploy-bare-metal.md [hunk 2]
```diff
@@ -63,30 +63,27 @@ For machines running a bookie and a Pulsar broker, more powerful machines are re
...
+   Bookie|0.2|256 MB|Journal: 8 GB<br><br />PD-SSDLedger: 16 GB, PD-STANDARD|Write throughput: 2 MB/s<br><br />Read throughput: 2 MB/s<br><br />|Write rate: 200 entries/s<br><br />Read rate: 200 entries/s
+   ZooKeeper|0.05|256 MB|Log: 8 GB, PD-SSD<br><br />Data: 2 GB, PD-STANDARD|/|/
+
+- The recommended hardware settings (**1000 Pulsar topics**)
+
+   Component | CPU|Memory|Storage|Throughput |Rate
+   |---|---|---|---|---|---
+   Broker|8|8 GB|/|Write throughput: 100 MB/s<br><br />Read throughput: 200 MB/s<br><br />|Write rate: 10,000 entries/s<br><br />Read rate: 20,000 entries/s
```
Comment (michaeljmarshall) [inline site2/docs/deploy-bare-metal.md:84]:
```suggestion
   Broker|8|8 GB|/|Write throughput: 100 MB/s<br><br />Read throughput: 200 MB/s<br><br />|Write rate: 10,000 messages/s<br><br />Read rate: 20,000 messages/s
```

File: site2/docs/deploy-bare-metal.md [hunk 3]
```diff
@@ -63,30 +63,27 @@ For machines running a bookie and a Pulsar broker, more powerful machines are re
...
-  - Broker
-    - CPU: 8
-    - Memory: 8GB
-  - Bookie
-    - CPU: 4
-    - Memory: 8GB
-    - Storage: 
-      - Journal: 256GB, PD-SSD
-      - Ledger: 2TB, PD-STANDARD
+A cluster consists of 3 broker nodes, 3 bookie nodes, and 3 ZooKeeper nodes. The following recommendation is suitable for one node.
```
Comment (Anonymitaet) [inline site2/docs/deploy-bare-metal.md:70]: @codelipenghui is this accurate?

**Comment by Anonymitaet** [comment]
> LGTM. @Anonymitaet - out of curiosity, how are you determining the expected throughput for clusters using this hardware? I ask because this kind of throughput may vary from version to version and based on the configurations of the cluster. While it's not necessary to publish this information, it would be interesting to how the throughput varies for each release of Pulsar.

@codelipenghui can you help answer this question? Thanks!

File: site2/docs/deploy-bare-metal.md (hunk 1 above)
Comment (codelipenghui) [reply site2/docs/deploy-bare-metal.md:76]: It should be entries because of the producer side batching, more entries rate consumes more CPU resources.

**Review by codelipenghui — APPROVED** [review]

File: site2/docs/deploy-bare-metal.md (hunk 1 above)
Comment (michaeljmarshall) [reply site2/docs/deploy-bare-metal.md:76]: Okay, I thought we always used the term messages, even if it meant we were describing batches of messages.

**Comment by michaeljmarshall** [comment]
> > LGTM. @Anonymitaet - out of curiosity, how are you determining the expected throughput for clusters using this hardware? I ask because this kind of throughput may vary from version to version and based on the configurations of the cluster. While it's not necessary to publish this information, it would be interesting to how the throughput varies for each release of Pulsar.
> 
> @codelipenghui can you help answer this question? Thanks!

@codelipenghui - any chance you can answer my additional questsions? Thanks!

**Pull request merged**